src/main/java/Jkbot/exception/JkBotException.java

```diff
@@ -0,0 +1,8 @@
+package Jkbot.exception;
+
+public class JkBotException extends Exception{
```

Comment:
Perhaps you can start tidying up the codes as per [coding standards](https://se-education.org/guides/conventions/java/intermediate.html) as I noticed this issue in several other files :)
```suggestion
public class JkBotException extends Exception {
```